examples/2D_flow_between_plates/case.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+import json
```

Comment:
are you sure this case is working as well as it was before you introduced the BC changes? can you run the same case with the master branch and compare the result? you did have to regenerate the goldenfiles it looks like...

Reply:
See above.